### DISCUSSION TITLE
A line code that unnecessary in CameraWebServer/app_httpd.cpp

### DISCUSSION THREAD

**Discussion by BackMountainDevil** [espressif/arduino-esp32#6289] [General]
[L22](https://github.com/espressif/arduino-esp32/blob/c4954dd582f5bde478316ddf184ded6979e812b3/libraries/ESP32/examples/Camera/CameraWebServer/app_httpd.cpp#L22)

The line is the same as L20. But that line has been  commented while this not. I test del L20 work fine.

```cpp
//#include "camera_index.h" // L20
#include "sdkconfig.h"
#include "camera_index.h" // L22
```

**Comment by VojtechBartoska** [comment]
@me-no-dev It seems the lines are just duplicated by mistake, right? I don't see any other reason for that.

**Reply by me-no-dev** [reply]
correct

**Reply by BackMountainDevil** [reply]
I think so. Because I am a green hand about this. I need a double check.

**Comment by VojtechBartoska** [comment]
Fix in #6370
